src/sentry/models/sentryshot.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model
+from sentry.db.models.base import region_silo_only_model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_only_model
+class SentryShot(Model):
+    """
+    Represents metadata for a screenshot within Sentry product
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
```

Comment:
I think you want this to be excluded - this table will have a lot of data, you don't really want this included in import/export (and this will fix the test complaints)